resources/overlay/usr/local/bin/fast_page_fault_helper.c

```diff
@@ -19,6 +19,14 @@
 
 #define MEM_SIZE_MIB (128 * 1024 * 1024)
 #define NANOS_PER_SEC 1000000000
+#define PAGE_SIZE 4096
+
+void touch_memory(void *mem, size_t size, char val) {
+    void *end = mem + size;
+    for (; mem < end; mem += PAGE_SIZE) {
+        *((char *)end) = val;
```

Comment:
we update `mem` over time, should be writing to it?

Reply:
Fixed (see also https://github.com/firecracker-microvm/firecracker/pull/5061#discussion_r1983729868)